ngi_reports/reports/project_summary.py

```diff
@@ -229,7 +229,29 @@ def __init__(self, config, LOG, working_dir, **kwargs):
             seq_template = "{}) Samples were sequenced on {} ({}) with a {} setup "\
                            "using {}. The Bcl to FastQ conversion was performed using {} from the CASAVA software "\
                            "suite. The quality scale used is Sanger / phred33 / Illumina 1.8+."
-            run_setup = fc_obj.get("run_setup")
+            ## Fetch run setup for the flowcell
+            run_setup = fc_obj.get("RunInfo").get('Reads')
+            ## Sort by the order of reads
+            run_setup = sorted(run_setup, key=lambda k: k['Number'])
+            run_setup_text = ''
+            read_count = 0
+            index_count = 0
+            for read in run_setup:
+                run_setup_text += read['NumCycles'].encode("utf-8")
```

Comment:
We don't need to encode it to utf-8, since we'll run it on python3

Reply:
Thx! Fixed! /chuan